src/mpcCoreKit.ts

```diff
@@ -310,6 +314,17 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     }
 
     // if not redirect flow or session rehydration, ask for factor key to login
+
+    // dont wait for wasm to be loaded, we can reload it during signing if not loaded
+    this._tssLib
+      .load()
+      .then((lib) => {
+        this.wasmLib = lib;
+        return true;
+      })
+      .catch((err: unknown) => {
```

Comment:
This should be thrown, if this were to fail it wouldn't be able to sign regardless and would likely fail on the second load for the same reason.

Reply:
so user might never use signing after init, that's why i m not throwing here, init function is used before login. User might just want to login to app and might never use signing so it better to not throw here.